tools/cli/adminClusterCommands.go

```diff
@@ -29,6 +29,9 @@ import (
 	"github.com/uber/cadence/common/types"
 )
 
+// An indirection for the prompt function so that it can be mocked in the unit tests
+var promptFn = prompt
+
```

Comment:
:| 

I mean, it works, but I/O is a dependency like anything else, we should just inject it somehow.  Though I'm not familiar with how much of a change that'd be to this code.